src/averageAges.js

```diff
@@ -38,6 +58,26 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+
+  const allFamale = people.filter(item => {
+    if (withChildren === undefined) {
+      return item.sex === 'f';
```

Comment:
Consider using variables with meaningful names instead of magic strings and numbers